pages/public_cloud/containers_orchestration/managed_kubernetes/installing-popeye/guide.de-de.md

```diff
@@ -393,21 +393,20 @@ Please enter your OpenStack Password:
 Now, you need to install the CLI tools:
 
 ```
-pip install python-openstackclient awscli awscli-plugin-endpoint
+pip install python-openstackclient awscli
 ```
 
 The command install `openstack client` to generate your credentials, `aws CLI` and mandatory `endpoint plugin for aws CLI`:
```

Comment:
> and mandatory `endpoint plugin for aws CLI`:

Reply:
Good catch, I removed this and added the missing `s` to `install` 👍